hknweb/academics/models/course_surveys.py

```diff
@@ -1,17 +1,32 @@
 from django.db import models
 from django.core.validators import MinValueValidator
+from django.db.models import Count, Case, When, Value
 
 from .base_models import AcademicEntity
+from .icsr import ICSR
 
 
 class Question(AcademicEntity):
-    pass
-
-
+    CHILDREN = ["rating"]
+    @staticmethod
+    def merge(primary, duplicates):

```

Comment:
This method is virtually the same as the `AcademicEntity` merge function, it's refactor so that we can use just a single merge function with class-specific variables?